platform/ui/src/components/AllInOneMenu/__stories__/allInOneMenu.stories.mdx

```diff
@@ -0,0 +1,137 @@
+import AllInOneMenu from '../AllInOneMenu';
+import AllInOneMenuItemPanel from '../AllInOneMenuItemPanel';
+import AllInOneMenuSubMenu from '../AllInOneMenuSubMenu';
+import AllInOneMenuItem from '../AllInOneMenuItem';
+import AllInOneMenuDividerItem from '../AllInOneMenuDividerItem';
+import AllInOneMenuHeaderItem from '../AllInOneMenuHeaderItem';
+import InputRange from '../../InputRange/index.js';
+import SwitchButton from '../../SwitchButton/index.js';
+
+import { ArgsTable, Story, Canvas, Meta } from '@storybook/addon-docs';
+import { createComponentTemplate } from '../../../storybook/functions/create-component-story';
+
+export const argTypes = {
+  component: AllInOneMenu,
+  title: 'Components/AllInOneMenu',
+};
+
+<Meta
+  title="Components/AllInOneMenu"
+  component={AllInOneMenu}
+/>
+
+export const AllInOneMenuTemplate = args => (
+  <div className="w-80">
+    <AllInOneMenu {...args}>
+      <AllInOneMenuItemPanel label="Panel 1">
+        <AllInOneMenuItem
+          label="Item 1"
+          key="0"
```

Comment:
It is the react key and is needed to avoid the verbose warnings you see in the console if it is excluded. https://react.dev/learn/rendering-lists#keeping-list-items-in-order-with-key